apps/mark/frontend/src/components/button_footer.tsx

```diff
@@ -1,35 +1,19 @@
-import styled, { css } from 'styled-components';
-
-const ButtonFooterButtons = css`
-  button {
-    padding-right: 10px;
-    padding-left: 10px;
-  }
-`;
+/* stylelint-disable order/properties-order */
```

Comment:
Getting in your way or just annoying?

Reply:
Not a particularly useful rule and I believe there's more standardisation around alphabetical ordering these days. I recently went down a rabbit hole of trying to update our stylelint import to the latest (which correctly enforces alphabetical ordering by default), but the fallout of that change was a bit larger than I was ready to take on 😅